<?xml version="1.0"?>
<launch>
  <!-- launches goal set under / namespace-->
  <group ns="drone7">
      <node pkg="carp_ros" type="goal_set.py" name="poseCaptain" output="screen">
      </node> 
  </group>
</launch>